benchmarking/README.md

```diff
@@ -160,82 +233,138 @@ Click on the `Run!` button. It will automatically start the process. Depending o
     - ```output_token_per_s_per_request``` bar plot: This plot shows the number of output tokens per second per request. One should see good agreement between the client and server-side metrics. For endpoints that support dynamic batching, one should see a decreasing trend in metrics as the batch size increases.
 
     - ```throughput_token_per_s``` bar plot: This plot shows the total tokens generated per second per batch. One should see good agreement between the client and server-side metrics. This metric represents the total number of tokens generated per second, which is the same as the previous metric for batch size = 1. However, for batch size > 1, it is estimated as the average of ```output_token_per_s_per_request * batch_size_used``` for each batch, to account for more tokens being generated due to concurrent requests being served in batch mode.
-#### Using terminal
 
-Users have this option if they want to experiment using values that are beyond the limits specified in the Streamlit app parameters.
+#### Performance on Chat
 
-1. Open the file `run.sh` and configure the following parameters in there:
+This option allows you to measure performance during a multi-turn conversation with an LLM. The interface should look like this:
 
-   - model: model name to be used. If it's a COE model, add "COE/" prefix to the name. Example: "COE/Meta-Llama-3-8B-Instruct"
-   - mean-input-tokens: average number of input tokens. It's recommended to choose no more than 2000 tokens to avoid long waitings. Default value: 1000.
-   - mean-output-tokens: average number of output tokens. It's recommended to choose no more than 2000 tokens to avoid long waitings. Default value: 1000.
-   - max-num-completed-requests: maximum number of completed requests. Default value: 32 
-   - num-concurrent-workers: number of concurrent workers. Default value: 1. 
-   - timeout: time when the process will stop. Default value: 600 seconds
-   - results-dir: path to the results directory. Default value: "./data/results/llmperf"
-   - additional-sampling-params: additional params for LLM. Default value: '{}'
+![perf_on_chat_image](./imgs/performance_on_chat.png)
 
-2. Run the following command in terminal. The performance evaluation process will start running and a progress bar will be shown until it is done.
+1. Enter a model name
 
-```
-sh run.sh
-```
+  - If the model specified in the `ai_starter-kit/.env` file is not a CoE model, you can skip this step.
```

Comment:
same suggestion as the one for synthetic perf eval

Reply:
✅